Aura instance must be accessible from the machine running `neo4j-admin` (#1664)
neo4j/docs-operations: modules/ROOT/pages/tools/neo4j-admin/push-to-cloud.adoc

This operation is secured and TLS encrypted end to end.
====

== Prerequisites

* The Neo4j Aura instance must already be provisioned and running.
* The Neo4j Aura instance must be accessible from the machine running `neo4j-admin`.
Otherwise, the upload will fail with SSL errors.
* Your local database must be stopped before you run the `push-to-cloud` command with the `--database` argument.

> The `push-to-cloud` function cannot be used with a source database that is currently in use.

Reviewer: ```suggestion
You cannot run the `push-to-cloud` command against a source database that is currently in use.
```
